module/main.js

```diff
@@ -1,6 +1,6 @@
 const axios = require('axios');
 
-const ban = (userId, userTag, databaseName, banReason, banProof) => {
+const ban = async (userId, userTag, databaseName, banReason, banProof) => {

```

Comment:
Convert to standardized async function(userId, userTag, databaseName, banReason, banProof) {}

Reply:
arrow functions can be unrecognized in older versions of NodeJS and can generically cause problems. I myself no longer use them for this reason, and seeing that this is an NPM module and it can go basically anywhere, I feel it would be better for them to be standardized callback functions.